code/__DEFINES/misc.dm

```diff
@@ -550,4 +550,4 @@ GLOBAL_LIST_INIT(pda_styles, sortList(list(MONO, VT, ORBITRON, SHARE)))
 //while using the SKILLCHIP_RESTRICTED_CATEGORIES flag
 #define SKILLCHIP_CATEGORY_GENERAL "general"
 #define SKILLCHIP_CATEGORY_JOB "job"
-#define SKILLCHIP_CATEGORY_FIREMAN_CARRYING "fireman carrying"
+#define SKILLCHIP_CATEGORY_SYNDICATE "syndicate"
```

Comment:
❓

Reply:
this was in Anne's code, she added it but it's not used anywhere. I kept it in case it was planned to be used, since defines are better than just strings